test/parallel/test-assert-typedarray-deepequal.js

```diff
@@ -99,6 +99,10 @@ suite('notEqualArrayPairs', () => {
         makeBlock(assert.deepStrictEqual, arrayPair[0], arrayPair[1]),
         assert.AssertionError
       );
+      assert.throws(
+        makeBlock(assert.partialDeepStrictEqual, arrayPair[0], arrayPair[1]),
+        assert.AssertionError
+      );
```

Comment:
Can you add a pair of `[new ArrayBuffer(3), new SharedArrayBuffer(3)]` and `[new SharedArrayBuffer(2), new ArrayBuffer(2)]` please?

Reply:
all the relevant tests have been added to `test-assert-objects.js`, including the ones you just mentioned. But I can add them here too if you want